go/vt/discovery/replicationlag.go

```diff
@@ -108,6 +108,20 @@ func filterByLag(tabletStatsList []*TabletStats) []*TabletStats {
 				replag: ts.Stats.SecondsBehindMaster})
 		}
 	}
+	if len(snapshots) == 0 {
+		// We get here if all tablets are over the high
+		// replication lag threshold, and theire lag is
+		// different enough that the 70% mean computation up
+		// there didn't find them all in a group. For
+		// instance, if *minNumTablets = 2, and we have two
+		// tablets with lag of 3h and 30h.
+		// In that case, we just use them all.
+		for _, ts := range list {
+			snapshots = append(snapshots, tabletLagSnapshot{
+				ts:     ts,
+				replag: ts.Stats.SecondsBehindMaster})
+		}
```

Comment:
I thought we should never return a tablet that's lagged by greater than the 'very high' threshold, because the data is too stale. This seems to contradict that.